src/SDKs/Automation/Management.Automation/Microsoft.Azure.Management.Automation.csproj

```diff
@@ -1,48 +1,25 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-	<!-- Please do not move/edit code below this line -->
-	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-	<!-- Please do not move/edit code below this line -->
-
-	<PropertyGroup>
-		<PackageId>Microsoft.Azure.Management.Automation</PackageId>
-		<Description>Provides Microsoft Azure Automation management operations including the ability to create, update and delete runbooks and schedules.</Description>
-		<AssemblyTitle>Microsoft Azure Automation Management Library</AssemblyTitle>
-		<Version>2.0.3</Version>
-		<AssemblyName>Microsoft.Azure.Management.Automation</AssemblyName>    
-		<PackageTags>Microsoft Azure Management Automation</PackageTags>
-		<PackageReleaseNotes/>
-	</PropertyGroup>
-	<PropertyGroup>
-		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
-	</PropertyGroup>
-
-	<ItemGroup>
-		<PackageReference Include="Hyak.Common" Version="1.1.3" />
-		<PackageReference Include="Microsoft.Azure.Common " Version="2.1.3" />
-		<PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="[1.6.0,2.0.0)" />
-	</ItemGroup>
-
-	<ItemGroup Condition=" '$(TargetFramework)' == 'net452' ">
-		<Reference Include="System" />
-		<Reference Include="Microsoft.CSharp" />
-	</ItemGroup>
-
-	<ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.5' ">
-		<PackageReference Include="Microsoft.NETCore.Platforms" Version="1.1.0" />
-		<PackageReference Include="System.Diagnostics.Debug" Version="4.3.0" />
-		<PackageReference Include="System.Diagnostics.Tools" Version="4.3.0" />
-		<PackageReference Include="System.Diagnostics.Tracing" Version="4.3.0" />
-		<PackageReference Include="Microsoft.CSharp" Version="4.0.1" />
-		<PackageReference Include="System.Threading.Thread" Version="4.0.0" />
-		<PackageReference Include="System.Threading.Tasks" Version="4.3.0" />
-		<PackageReference Include="System.Threading.Timer" Version="4.3.0" />
-		<PackageReference Include="System.Collections" Version="4.3.0" />
-		<PackageReference Include="System.Linq" Version="4.3.0" />
-		<PackageReference Include="System.Runtime" Version="4.3.0" />
-		<PackageReference Include="System.Net.Http" Version="4.3.0" />
-	</ItemGroup>
-
-	<!-- Please do not move/edit code below this line -->
-	<Import Condition="Exists('[MSBuild]::GetPathOfFileAbove(AzSdk.RP.props)')" Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
-	<!-- Please do not move/edit code above this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
```

Comment:
@vrdmr this file has not been merged properly. It does not have the upstream changes.

Reply:
I merged the changes with the upstream. I have changed the csproj to remove any dependencies like the following - 
```
<ItemGroup>
		<PackageReference Include="Hyak.Common" Version="1.1.3" />
		<PackageReference Include="Microsoft.Azure.Common " Version="2.1.3" />
		<PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="[1.6.0,2.0.0)" />
	</ItemGroup>
```

The new csproj file removes a lot from existing csproj.